scripts/o3de/o3de/enable_gem.py

```diff
@@ -102,6 +105,27 @@ def enable_gem_in_project(gem_name: str = None,
     # Convert each path to pathlib.Path object and filter out duplicates using dict.fromkeys
     buildable_gems = list(dict.fromkeys(map(lambda gem_path_string: pathlib.Path(gem_path_string), buildable_gems)))
 
+    # check compatibility
+    if force:
+        logger.warning(f'Bypassing version compatibility check for {gem_json_data["gem_name"]}.')
+    else:
+        incompatible_objects = compatibility.get_gem_project_incompatible_objects(gem_json_data, project_path, check=check, gem_paths=buildable_gems)
+        if check:
```

Comment:
The force option warning shouldn't occur if the `check` option has been specified.

Reply:
`check` and `force` are set as mutually exclusive (add_mutually_exclusive_group) so we shouldn't have both set at the same time.

Reply:
Thanks for making the options mutually exclusive.